client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -91,6 +142,14 @@ const EditPage = createClass({
 		document.removeEventListener('keydown', this.handleControlKeys);
 	},
 
+	editTypeIsNew : function() {
+		return (this.editType == 'new');
```

Comment:
There is no real reason to create an identical variable like above that I can see:
```
if(this.props.editType == type) {
	this.editType = this.props.editType;
};
```

`editType` should never change while the user is viewing the page. If they save a `new` brew, the page redirects to `/edit` which will give us a new editType, but we aren't ever changing it within the page itself. For constant settings like this, it is preferred to just check `this.props` directly.

Reply:
`this.editType` was only created as a separate entity to hold the result of the `this.props.editType` validation. Now that it's been/being removed, we can shift to using `this.props.editType` for exactly the reasons you've outlined.

Reply:
Perfect.